src/MeetingsSDKAdapter.js

```diff
@@ -196,7 +197,7 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
    */
   createMeeting(target) {
     return from(this.datasource.meetings.create(target)).pipe(
-      flatMap(({id, sipUri}) => from(this.fetchMeetingTitle(sipUri)).pipe(map((title) => ({ID: id, title})))),
+      flatMap(({id}) => from(this.fetchMeetingTitle(target)).pipe(map((title) => ({ID: id, title})))),
```

Comment:
Can you rename the argument from `target` to `destination`?
Previously we couldn't do that because we were destructing the attribute from SDK meeting object with the name, but that's no longer the case. I think it makes it more readable as it is consistent with the wording in other places.